BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -274,15 +316,86 @@ public static Vector3 GetAirToAirFireSolution(MissileBase missile, Vessel target
             }
 
             return targetPosition;
-        }
 
+            /* //if should unguided missiles require more accurate firing solution. Test with above inverse quadratic equation for lead time first, as this gets called by both missilefire and the AI, so that's duplicate trajectory sims otherwise
```

Comment:
Presumably this isn't needed if the above works.

Reply:
Correct.